exchanges/gemini/gemini_test.go

```diff
@@ -7,235 +7,188 @@ import (
 
 	"github.com/gorilla/websocket"
 	"github.com/thrasher-corp/gocryptotrader/common"
-	"github.com/thrasher-corp/gocryptotrader/config"
 	"github.com/thrasher-corp/gocryptotrader/currency"
 	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/websocket/wshandler"
 )
 
 // Please enter sandbox API keys & assigned roles for better testing procedures
-
 const (
-	apiKey1           = ""
-	apiSecret1        = ""
-	apiKeyRole1       = ""
-	sessionHeartBeat1 = false
-
-	apiKey2           = ""
-	apiSecret2        = ""
-	apiKeyRole2       = ""
-	sessionHeartBeat2 = false
-
+	apiKey                  = ""
+	apiSecret               = ""
+	apiKeyRole              = ""
+	sessionHeartBeat        = false
 	canManipulateRealOrders = false
 )
 
-func TestAddSession(t *testing.T) {
-	var g1 Gemini
-	if Session[1] == nil {
-		err := AddSession(&g1, 1, apiKey1, apiSecret1, apiKeyRole1, true, true)
-		if err != nil {
-			t.Error("Test failed - AddSession() error", err)
-		}
-		err = AddSession(&g1, 1, apiKey1, apiSecret1, apiKeyRole1, true, true)
-		if err == nil {
-			t.Error("Test failed - AddSession() error", err)
-		}
-	}
-
-	if len(Session) <= 1 {
-		var g2 Gemini
-		err := AddSession(&g2, 2, apiKey2, apiSecret2, apiKeyRole2, false, true)
-		if err != nil {
-			t.Error("Test failed - AddSession() error", err)
-		}
-	}
-}
-
-func TestSetDefaults(t *testing.T) {
-	Session[1].SetDefaults()
-	Session[2].SetDefaults()
-}
-
-func TestSetup(t *testing.T) {
-	cfg := config.GetConfig()
-	cfg.LoadConfig("../../testdata/configtest.json")
-	geminiConfig, err := cfg.GetExchangeConfig("Gemini")
-	if err != nil {
-		t.Error("Test Failed - Gemini Setup() init error")
-	}
-
-	geminiConfig.AuthenticatedWebsocketAPISupport = true
-	geminiConfig.AuthenticatedAPISupport = true
-	geminiConfig.Websocket = true
-	Session[1].Setup(&geminiConfig)
-	Session[2].Setup(&geminiConfig)
-
-	Session[1].APIKey = apiKey1
-	Session[1].APISecret = apiSecret1
-
-	Session[2].APIKey = apiKey2
-	Session[2].APISecret = apiSecret2
-
-	Session[1].APIUrl = geminiSandboxAPIURL
-	Session[2].APIUrl = geminiSandboxAPIURL
-}
+var g Gemini
 
 func TestGetSymbols(t *testing.T) {
 	t.Parallel()
-	_, err := Session[1].GetSymbols()
+	_, err := g.GetSymbols()
 	if err != nil {
 		t.Error("Test Failed - GetSymbols() error", err)
 	}
 }
 
 func TestGetTicker(t *testing.T) {
 	t.Parallel()
-	_, err := Session[2].GetTicker("BTCUSD")
+	_, err := g.GetTicker("BTCUSD")
 	if err != nil {
 		t.Error("Test Failed - GetTicker() error", err)
 	}
-	_, err = Session[1].GetTicker("bla")
+	_, err = g.GetTicker("bla")
 	if err == nil {
 		t.Error("Test Failed - GetTicker() error", err)
 	}
 }
 
 func TestGetOrderbook(t *testing.T) {
 	t.Parallel()
-	_, err := Session[1].GetOrderbook("btcusd", url.Values{})
+	_, err := g.GetOrderbook("btcusd", url.Values{})
```

Comment:
Can create a const testCurrency = "btcusd" for here and below as it is reused quite a bit